app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -7,36 +7,24 @@ def self.miq_template_type
       parent::Parser::ParserDictionaryConstants::MIQ_TYPES["template"]
     end
 
-    def initialize(ems, _options = nil)
-      ems_auth = ems.authentications.first
-
-      @ems        = ems
-      @connection = ems.connect(:user => ems_auth.userid,
-                                :pass => ems_auth.password,
-                                :host => ems.endpoints.first.hostname,
-                                :port => ems.endpoints.first.port)
-      @parser     = init_parser(@connection)
+    def initialize(connection, _options = nil)
+      @connection = connection
+      @parser     = init_parser
     end
 
     def ems_inv_to_hashes
-      log_header = "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS : [#{@ems.name}] id: [#{@ems.id} ref: #{@ems.uid_ems}]"
-
-      $log.info("#{log_header}...")
```

Comment:
Why remove these logs?  They are actually extremely useful for debugging by support.